src/docs/style/themes.mdx

```diff
@@ -22,42 +27,97 @@ One of the key benefits of PX Blue is consistency of look & feel between differe
+</div>
 
-The PX Blue themes packages includes two distinct themes:
+# Blue Theme / Light Theme
 
--   **Blue** is the default theme for all PX Blue applications. It should be applied at the root level of your app. It provides a variety of shades of blue for use in buttons, toolbars, etc. The standard background color is a light gray.
--   **Blue Dark** is complementary to the Blue theme. It is intended primarily for applications used in low-light conditions. The primary colors used in the Blue Dark theme are various shades of gray with blue accents. It can be used as a standalone theme or combined with the Blue (default) theme to provide contrast for areas such as detail sidebars.
+<div style={REGULAR_WIDTH_STYLE}>
+    <ImageGrid images={[LightThemeExample]} regularWidth caption={'An example of a light-themed application.'} />
+</div>
 
-<DemoCard
-    repository={'showcase-demo'}
-    angular={`https://stackblitz.com/github/pxblue/angular-showcase-demo/tree/master`}
-    react={`https://codesandbox.io/s/github/pxblue/react-showcase-demo/tree/master`}
-/>
+The blue theme is the default theme for all PX Blue applications. With a clean color vibrance and high text legibility, it is designed to suit day-to-day use on a computer monitor, on-site facility inspection with a tablet, and on a mobile phone screen. It should be applied at the root level of your app.
 
-> **NOTE:** Combining multiple themes in your application is optional. You may stick to using a single theme (Blue) for your entire application and then apply one-off styling where necessary to achieve other effects.
+The light theme uses our iconic [Blue 500](/style/color) color (`#007bc1`) throughout the application, notably in toolbars, buttons, and icons. In some places, Light Blue 500 is applied as an accent/secondary color on interactive UI elements such as form controls. The light theme's base background color is White 200, although the surface elements are often shaded in White 50 (pure white).
 
+<ImageGrid images={[LightThemeColors]} regularWidth />
+<Divider />
+
+# Dark Theme / Blue Dark Theme
+
+<div style={REGULAR_WIDTH_STYLE}>
+    <ImageGrid images={[DarkThemeExample]} regularWidth caption={'An example of a dark-themed application.'} />
+</div>
+
+The dark theme is intended primarily for applications used in low-light conditions. As the name indicates, it utilizes different shades of dark colors (grays and blacks) and uses other colors only for interactive components and status displays.
+
+<ImageGrid images={[DarkThemeColors]} regularWidth />
+
+To accommodate for [accessibility requirements](/design/accessibility), PX Blue uses the 200 shade of colors when text is used on a dark background, such as in an outlined button and a text field label. For "contained" elements, such as in an elevated / contained button, the same style is used as in the light theme (500 shades).
+
+<div style={REGULAR_WIDTH_STYLE}>
+    <ImageGrid
+        images={[OutlinedAndFlat]}
```

Comment:
I don't think docit has any images with green _bars_ in them — the idea was so that people with color blindness can single them out better... Although I guess there are better ways to handle that.

There certainly are images with green _backgrounds_. However, a green background would not work here, as the dark background is critical in illustrating my point.

Reply:
Ah, okay, I think I must be misremembering. I thought we used red and green bars, but if we're not, then ignore me.